src/main/java/hello/NotificationController.java

```diff
@@ -22,16 +24,14 @@
     @SendTo("/topic/myscores")
     public String handleRequest(HttpServletRequest request, HttpServletResponse response)
     {
-        final List<Batsman> list = new ArrayList<>();
-        final Batsman batsmanA = new Batsman();
-        batsmanA.setRuns(Integer.parseInt(request.getParameter("runs-a")));
-        batsmanA.setBalls(Integer.parseInt(request.getParameter("balls-a")));
-        list.add(batsmanA);
-        final Batsman batsmanB = new Batsman();
-        batsmanB.setRuns(Integer.parseInt(request.getParameter("runs-b")));
-        batsmanB.setBalls(Integer.parseInt(request.getParameter("balls-b")));
-        list.add(batsmanB);
-        template.convertAndSend("/topic/myscores", list);
+        final List<MessengerNotification> notificationList = new ArrayList<>();
+        final MessengerNotification notificationA = new MessengerNotificationBuilder()
+                .withMessage(request.getParameter("message"))
+                .withSenderName(request.getParameter("sender-name"))
+                .withReceiverName(request.getParameter("receiver-name"))
```

Comment:
[minor] You might want to use date here to show that our app covers dates also :P

Reply:
This should be done on server side. It's in the processing notification task and it rightfully belongs to @Bzdeco.